tests/Integration/Behaviour/Features/Context/Domain/Product/CommonProductFeatureContext.php

```diff
@@ -379,8 +379,7 @@ private function assertProductType(string $productReference, string $productType
         $productId = $this->getSharedStorage()->get($productReference);
         $product = new Product($productId);
         Assert::assertEquals($productTypeName === ProductType::TYPE_VIRTUAL, (bool) $product->is_virtual);
-        // cache_is_pack is automatically updated by legacy code when removing all pack items so it's not worth testing it for now
-        // Assert::assertEquals($productTypeName === ProductType::TYPE_PACK, (bool) $product->cache_is_pack);
+        Assert::assertEquals($productTypeName === ProductType::TYPE_PACK, (bool) $product->cache_is_pack);
```

Comment:
:+1:

Reply:
Yeah it didn't work so I actually fixed it ^^